proto/api/v1/api.proto

```diff
@@ -111,7 +111,7 @@ message Processor {
 
   // -- immutable fields -------------------------------------------------------
 
-  string name = 4 [(google.api.field_behavior) = IMMUTABLE];
+  string type = 4 [(google.api.field_behavior) = IMMUTABLE];
```

Comment:
For future reference - we are changing the field name which will change the JSON field name and thus it's a breaking change. Old gRPC clients should still work though, because the binary representation of the message should be the same. We are aware of this and are ignoring the validation warning on purpose, the only affected client right now should be the UI and we will change it when we move to the new proto definition (that's not done in this PR).